libs/knowledge-store/ragstack_knowledge_store/graph_store.py

```diff
@@ -118,7 +111,7 @@ def __init__(
         embedding: EmbeddingModel,
         *,
         node_table: str = "graph_nodes",
-        edge_table: str = "graph_edges",
+        targets_table: str = "graph_targets",
```

Comment:
This is a breaking change

Reply:
Yes.

Reply:
Yeah. Different schema too.

Reply:
Then we need to cut the release with this change before the LangChain PR is merged.

Reply:
Generally agree we should get this in (as well as the one I have started on after this to actually serialize metadata) since they are schema changes. That said, I don't know that we won't be able to make these changes after the LangChain PR is merged -- changes will be necessary, and we can't treat that as a point after which we can never make changes.